src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -313,10 +321,51 @@ export function InternalDashboardTopNav({
       });
     }
     if (showWriteControls && managed) {
-      allBadges.push(getManagedContentBadge(dashboardManagedBadge.getTooltip()));
+      const badgeProps = {
+        ...getManagedContentBadge(dashboardManagedBadge.getManagedText(), false),
+        onClickAriaLabel: dashboardManagedBadge.getAriaLabel(),
+        iconOnClick: () => setIsPopoverOpen(!isPopoverOpen),
+        iconOnClickAriaLabel: dashboardManagedBadge.getAriaLabel(),
+        onClick: () => setIsPopoverOpen(!isPopoverOpen),
+      } as TopNavMenuBadgeProps;
+      allBadges.push({
+        renderCustomBadge: ({ badgeText }) => {
+          const badgeButton = <EuiBadge {...badgeProps}>{badgeText}</EuiBadge>;
+          return (
+            <EuiPopover
+              button={badgeButton}
+              isOpen={isPopoverOpen}
+              closePopover={() => setIsPopoverOpen(false)}
+              className="eui-hideFor--s eui-hideFor--xs"
```

Comment:
This badge **isn't** hidden on smaller viewports on `main` - so I think we should keep that behaviour. This is what it does on `main`: 

![image](https://github.com/user-attachments/assets/17990283-d514-4401-84f8-396c11119c4d)

Reply:
Good catch 👍🏻 , sorry! Added the needed classnames to the badge itself!

Reply:
As far as I know, we shouldn't have these classnames at all - since, on main, the badge doesn't seem to disappear on smaller screens. Does that make sense?